ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1093,9 +1093,6 @@ periodics:
   - base_ref: main
     org: openshift
     repo: operator-framework-operator-controller
-  - base_ref: main
-    org: openshift
-    repo: operator-framework-catalogd
```

Comment:
It seems that the error to run `make jobs` is related to this change but how we should do it?